test/e2e/federation-ingress.go

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
For reference, see the title of this doc here - https://github.com/kubernetes/kubernetes/pull/29793